### PR TITLE
Only print RADIO_PARAM_PAN_ID at startup if the radio supports it

### PR DESCRIPTION
**Pull request by g-oikonomou**
In prop mode, the CC13xx/CC26xx radio does not do any h/w frame filtering. For this reason, the respective RF driver does not support `RADIO_PARAM_PAN_ID` (and a number of other things, such as `RADIO_PARAM_16BIT_ADDR`).

As a result, all images that use prop mode will print junk at startup. Basically, the line below in `platform_init_stage_three()` uses variable `pan` uninitialised.

```
  LOG_INFO(" RF: Channel %d, PANID 0x%04X\n", chan, pan);
```

This pull tidies up the function, initialises variables and makes sure to only print PAN ID if `NETSTACK_RADIO.get_value()` succeeds.

Reported by @m-calvo (Can you test this please?)

### CODE REVIEW

File: arch/platform/srf06-cc26xx/platform.c [hunk 1]
```diff
@@ -198,7 +198,13 @@ platform_init_stage_three()
           ti_lib_chipinfo_chip_family_is_cc13xx() == true ? "Yes" : "No",
           ti_lib_chipinfo_supports_ble() == true ? "Yes" : "No",
           ti_lib_chipinfo_supports_proprietary() == true ? "Yes" : "No");
-  LOG_INFO(" RF: Channel %d, PANID 0x%04X\n", chan, pan);
+  LOG_INFO(" RF: Channel %d", chan);
+
+  if(NETSTACK_RADIO.get_value(RADIO_PARAM_PAN_ID, &pan) == RADIO_RESULT_OK) {
+    LOG_INFO(", PANID 0x%04X\n", pan);
```
Comment (simonduq) [inline arch/platform/srf06-cc26xx/platform.c]: you probably want `LOG_INFO_` here (not to prepend the log prefix)

File: arch/platform/srf06-cc26xx/platform.c [hunk 2]
```diff
@@ -198,7 +198,13 @@ platform_init_stage_three()
...
+  }
+  LOG_INFO("\n");
```
Comment (simonduq) [inline arch/platform/srf06-cc26xx/platform.c]: you probably want `LOG_INFO_` here (not to prepend the log prefix)

Comment (simonduq) [reply arch/platform/srf06-cc26xx/platform.c]: shouldn't this go to `else` to avoid printing `\n` twice?

Comment (g-oikonomou) [reply arch/platform/srf06-cc26xx/platform.c]: Argh I never meant for the PANID line to have `\n` will fix in a moment

Comment (g-oikonomou) [reply arch/platform/srf06-cc26xx/platform.c]: That's done now and rebased

File: arch/platform/srf06-cc26xx/platform.c [hunk 3]
```diff
@@ -209,7 +209,13 @@ platform_init_stage_three()
           ti_lib_chipinfo_chip_family_is_cc13xx() == true ? "Yes" : "No",
           ti_lib_chipinfo_supports_ble() == true ? "Yes" : "No",
           ti_lib_chipinfo_supports_proprietary() == true ? "Yes" : "No");
-  LOG_INFO(" RF: Channel %d, PANID 0x%04X\n", chan, pan);
+  LOG_INFO(" RF: Channel %d", chan);
+
+  if(NETSTACK_RADIO.get_value(RADIO_PARAM_PAN_ID, &pan) == RADIO_RESULT_OK) {
+    LOG_INFO(", PANID 0x%04X", pan);
```
Comment (simonduq) [inline arch/platform/srf06-cc26xx/platform.c]: I still think you want `LOG_INFO_` here instead (note the final `_`)

Comment (g-oikonomou) [reply arch/platform/srf06-cc26xx/platform.c]: Still not used to this... So I need `INFO_` here as well as in the final `\n`

Comment (simonduq) [reply arch/platform/srf06-cc26xx/platform.c]: Yes, and actually also the "RF: channel one". You need it whenever continuing a previous line of log. Else a prefix will added every time

Comment (g-oikonomou) [reply arch/platform/srf06-cc26xx/platform.c]: Yep. The "RF: Channel" is at the start of a line.

**Pull request merged**